server/system.go

```diff
@@ -44,6 +44,7 @@ func (s *Server) registerSystemRoutes(g *echo.Group) {
 			AllowSignUp:        false,
 			IgnoreUpgrade:      false,
 			DisablePublicMemos: false,
+			MaxUploadSizeMiB:   32,
```

Comment:
The default value is 32. So we don't need to add 

* store/db/migration/prod/0.12/06__default_upload_limit.sql
* store/db/seed/10005__system_setting.sql

Reply:
done

Reply:
> The default value is 32. So we don't need to add
> 
> * store/db/migration/prod/0.12/06__default_upload_limit.sql
> * store/db/seed/10005__system_setting.sql

There is a bug in https://github.com/usememos/memos/blob/main/server/resource.go#L73 , because the default value of MaxUploadSizeMiB isn't set in the database.

Reply:
> 

Indeed.

In this case, my initial approach to seed and migrate the database was a little better.

In databases without the setting "max-upload-size-mib", the backend will return "File size exceeds allowed limit of 0 MiB".

Changing the default value "32" to something else in memos "System" section solves this, as it will create the missing setting key.

Simply changing that "0" parameter in https://github.com/usememos/memos/blob/main/server/resource.go#L73 to "32" works, but this breaks server enforcing the setting if uploads are explicitly disabled by setting max size to "0".

As this can be perceived as security issue (filling all disk space would break server functionality), I will look into this, hoping to find a better solution.